packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/TypedSnippets/ModelReaderWriterOptionsSnippet.cs

```diff
@@ -4,14 +4,19 @@
 using System.ClientModel.Primitives;
 using Microsoft.Generator.CSharp.ClientModel.Providers;
 using Microsoft.Generator.CSharp.Expressions;
+using Microsoft.Generator.CSharp.Providers;
 using Microsoft.Generator.CSharp.Snippets;
+using static Microsoft.Generator.CSharp.Snippets.Snippet;
 
 namespace Microsoft.Generator.CSharp.ClientModel.Snippets
 {
     internal sealed record ModelReaderWriterOptionsSnippet(ValueExpression Untyped) : TypedSnippet<ModelReaderWriterOptions>(Untyped)
     {
         public static readonly ModelReaderWriterOptionsSnippet Wire = ModelSerializationExtensionsProvider.Instance.WireOptions;
-
         public ValueExpression Format => new MemberExpression(this, nameof(ModelReaderWriterOptions.Format));
+        internal static StringSnippet WireFormat => Literal("W");
+        internal static StringSnippet JsonFormat => Literal("J");
+        internal static ModelReaderWriterOptionsSnippet MrwNullableOptions(ParameterProvider param) => new(param);
```

Comment:
The name says nullable options?  How does this create a nullable options snippet?